Release notes 10.1

jameshupp opened the pull request:
## Changes
Adds new release notes to updates

## Notes
I’m not sure what to link to for the stemcell update. If someone points me to something I’ll be glad to add that.

Commit:
initial commit of new release notes

might have to make a few minor edits upcoming

mogul reviewed (dismissed):
Looks good to me. I don't know that there's any great link for the stemcell; it's more important to list the CVEs. I'm approving here but not merging to give others a chance to weigh in. I'd say if you hadn't heard from anyone else in the next couple hours, go ahead and merge!

Commit:
deleting previous list of USNs

Commit:
Merge branch 'master' into update-20170503

Commit:
closely associate upgrades and USNs

Before it was a list of one and a list of another. Now they’re more
P->Q, which should help users understand the importance of using the
most recent versions.

Commit:
Merge branch 'master' into update-20170503
